### PR TITLE
Translate the newly added content of results.yml into Traditional Chinese

### PR DESCRIPTION
**Pull request by mingjunlu**
- Update the `zh-Hant` translator list
- Update `zh-Hant` translation of `results.yml`

### CODE REVIEW

**Comment by mingjunlu** [comment]
@ymcheung Would you please review my PR? Many thanks 🙇‍♂️

**Comment by ymcheung** [comment]
> @ymcheung Would you please review my PR? Many thanks 🙇‍♂️
Sure, I also have some translation accents, you can refer to my suggestion and the decision is yours.

File: src/i18n/zh-Hant/results.yml [hunk 1]
```diff
@@ -459,3 +471,39 @@ translations:
...
+      t: >
+          每一區上方的 <span class="hint-export-button">匯出</span> 按鈕可以讓你取得圖表的原始 JSON 資料或 GraphQL query。
+
+    - key: hints.completion_hint
+      t: >
+          所有問題都是選填，所以有些問題的回答數可能會比較少。這個圖示（<span class="hint-completion-indicator">◕</span>）會註明回答問題的人有多少。
+
+    - key: hints.tool_modal_hint
+      t: >
+          點選某個技術的名稱，你就能得到更多資訊，並深入檢視相關資料。
```
Comment (ymcheung) [inline src/i18n/zh-Hant/results.yml:497]: 點選技術的名稱後，可以看到更多細節，...

File: src/i18n/zh-Hant/results.yml [hunk 2]
```diff
@@ -459,3 +471,39 @@ translations:
...
+      t: >
+          每一區右上角的分段控制器（segmented control）可以切換顯示百分比或原始次數。
+
+    - key: hints.export_hint
+      t: >
+          每一區上方的 <span class="hint-export-button">匯出</span> 按鈕可以讓你取得圖表的原始 JSON 資料或 GraphQL query。
+
+    - key: hints.completion_hint
+      t: >
+          所有問題都是選填，所以有些問題的回答數可能會比較少。這個圖示（<span class="hint-completion-indicator">◕</span>）會註明回答問題的人有多少。
```
Comment (ymcheung) [inline src/i18n/zh-Hant/results.yml:493]: - 原文有更新？

問題都是選填，所以有些的回答數可能比較少。完成比例圖示 ... 會註明確切的答題人數。

File: src/i18n/zh-Hant/results.yml [hunk 3]
```diff
@@ -459,3 +471,39 @@ translations:
...
+
+    - key: hints.tool_modal_hint
+      t: >
+          點選某個技術的名稱，你就能得到更多資訊，並深入檢視相關資料。
+
+    - key: hints.rankings_modes_hint
+      t: >
+          你可以自由勾選排名圖表是否顯示滿意度、有興趣、實際使用、有聽過比例，使各類別的排名一目了然。
```
Comment (ymcheung) [inline src/i18n/zh-Hant/results.yml:501]: ~你可以~

File: src/i18n/zh-Hant/results.yml [hunk 4]
```diff
@@ -459,3 +471,39 @@ translations:
...
+
+    - key: hints.sharing_hint
+      t: >
+          科學研究顯示，透過 Twitter、Facebook，或 email 分享本研究調查，最高可以讓你的 JavaScript 功力增加 15%。
```
Comment (ymcheung) [inline src/i18n/zh-Hant/results.yml:505]: ... Facebook 或 Email ...

File: src/i18n/zh-Hant/results.yml [hunk 5]
```diff
@@ -459,3 +471,39 @@ translations:
...
+
+    - key: hints.tshirt_hint
+      t: >
+          另一項科學研究發現：購買我們的 T-shirt 可以讓你的 JavaScript 知識增加 9000 分！
```
Comment (ymcheung) [inline src/i18n/zh-Hant/results.yml:509]: 可以讓 JavaScript 知識 ...

**Review by ymcheung — APPROVED** [review]

File: src/i18n/zh-Hant/results.yml (hunk 2 above)
Comment (mingjunlu) [reply src/i18n/zh-Hant/results.yml:493]: 我當初是看到這邊有新的文檔所以就翻了 🤔
https://github.com/StateOfJS/state-of-js-graphql-results-api/pull/106/commits/87662d8cf89ba36c2fad8eaf9f3c7e7cf32b3fdd#diff-49c7375a7db640ad3432ec7e84a232f947f49704882888646200b2cce3f40041R503-R506

**Comment by mingjunlu** [comment]
It seems that @SachaG has already merged this pull request. I'll open a new one after adjusting these.

File: src/i18n/zh-Hant/results.yml [hunk 6]
```diff
@@ -459,3 +471,39 @@ translations:
...
     - key: award.most_write_ins_award.description
       t: 頒發給最多自行填答的項目。
+
+    ###########################################################################
+    # Hints
+    ###########################################################################
+
+    - key: hints.hint_hint
+      t: >
+          你在整份問卷調查結果都能看到像這樣的小提示，它會顯示額外的功能和細節。
```
[the remark this answers is not in the document]
Comment (mingjunlu) [reply src/i18n/zh-Hant/results.yml:481]: 通順多了，感謝提點 🙏

**Comment by SachaG** [comment]
Oh sorry if I merged too fast!

**Comment by mingjunlu** [comment]
Not a big deal! We're just proofreading some of the sentences 😊

File: src/i18n/zh-Hant/results.yml [hunk 7]
```diff
@@ -443,7 +455,7 @@ translations:
     - key: award.tool_interest_award.title
       t: 最有興趣
     - key: award.tool_interest_award.description
-      t: 頒發給開發者們最有興趣學習的技術。
+      t: 頒發給開發者們聽過後最有興趣學習的技術。
```
[the remark this answers is not in the document]
Comment (mingjunlu) [reply src/i18n/zh-Hant/results.yml:458]: 中文好像比較少用「名詞 + 們」，所以我把「們」也順便拿掉了
https://github.com/StateOfJS/state-of-js-graphql-results-api/pull/121/commits/4f370eb8f4fd43409228f822f340f7c8103b875a#diff-cc015bc94e16ef0a22a18fb5bec949b1eb4ddca6326461e44289d98ab9aa17e5R458

File: src/i18n/zh-Hant/results.yml (hunk 6 above)
[the remark this answers is not in the document]
Comment (mingjunlu) [reply src/i18n/zh-Hant/results.yml:481]: https://github.com/StateOfJS/state-of-js-graphql-results-api/pull/121/commits/4f370eb8f4fd43409228f822f340f7c8103b875a#diff-cc015bc94e16ef0a22a18fb5bec949b1eb4ddca6326461e44289d98ab9aa17e5R481

File: src/i18n/zh-Hant/results.yml [hunk 8]
```diff
@@ -459,3 +471,39 @@ translations:
...
+    # Hints
+    ###########################################################################
+
+    - key: hints.hint_hint
+      t: >
+          你在整份問卷調查結果都能看到像這樣的小提示，它會顯示額外的功能和細節。
+
+    - key: hints.units_switcher_hint
+      t: >
+          每一區右上角的分段控制器（segmented control）可以切換顯示百分比或原始次數。
```
[the remark this answers is not in the document]
Comment (mingjunlu) [reply src/i18n/zh-Hant/results.yml:485]: 已調整：
https://github.com/StateOfJS/state-of-js-graphql-results-api/pull/121/commits/4f370eb8f4fd43409228f822f340f7c8103b875a#diff-cc015bc94e16ef0a22a18fb5bec949b1eb4ddca6326461e44289d98ab9aa17e5R485

File: src/i18n/zh-Hant/results.yml [hunk 9]
```diff
@@ -459,3 +471,39 @@ translations:
...
+
+    - key: hints.export_hint
+      t: >
+          每一區上方的 <span class="hint-export-button">匯出</span> 按鈕可以讓你取得圖表的原始 JSON 資料或 GraphQL query。
```
[the remark this answers is not in the document]
Comment (mingjunlu) [reply src/i18n/zh-Hant/results.yml:489]: 已調整：
https://github.com/StateOfJS/state-of-js-graphql-results-api/pull/121/commits/4f370eb8f4fd43409228f822f340f7c8103b875a#diff-cc015bc94e16ef0a22a18fb5bec949b1eb4ddca6326461e44289d98ab9aa17e5R489

File: src/i18n/zh-Hant/results.yml (hunk 1 above)
Comment (mingjunlu) [reply src/i18n/zh-Hant/results.yml:497]: 已調整：
https://github.com/StateOfJS/state-of-js-graphql-results-api/pull/121/commits/4f370eb8f4fd43409228f822f340f7c8103b875a#diff-cc015bc94e16ef0a22a18fb5bec949b1eb4ddca6326461e44289d98ab9aa17e5R497

File: src/i18n/zh-Hant/results.yml (hunk 2 above)
Comment (mingjunlu) [reply src/i18n/zh-Hant/results.yml:493]: 調整後版本：
https://github.com/StateOfJS/state-of-js-graphql-results-api/pull/121/commits/4f370eb8f4fd43409228f822f340f7c8103b875a#diff-cc015bc94e16ef0a22a18fb5bec949b1eb4ddca6326461e44289d98ab9aa17e5R493

File: src/i18n/zh-Hant/results.yml (hunk 3 above)
Comment (mingjunlu) [reply src/i18n/zh-Hant/results.yml:501]: 已調整：
https://github.com/StateOfJS/state-of-js-graphql-results-api/pull/121/commits/4f370eb8f4fd43409228f822f340f7c8103b875a#diff-cc015bc94e16ef0a22a18fb5bec949b1eb4ddca6326461e44289d98ab9aa17e5R501

File: src/i18n/zh-Hant/results.yml (hunk 4 above)
Comment (mingjunlu) [reply src/i18n/zh-Hant/results.yml:505]: 已把「或」拿掉。不過考慮到 email 不是專有名詞，所以我選擇保留原本的小寫：
https://github.com/StateOfJS/state-of-js-graphql-results-api/pull/121/commits/4f370eb8f4fd43409228f822f340f7c8103b875a#diff-cc015bc94e16ef0a22a18fb5bec949b1eb4ddca6326461e44289d98ab9aa17e5R505

File: src/i18n/zh-Hant/results.yml (hunk 5 above)
Comment (mingjunlu) [reply src/i18n/zh-Hant/results.yml:509]: 已調整：
https://github.com/StateOfJS/state-of-js-graphql-results-api/pull/121/commits/4f370eb8f4fd43409228f822f340f7c8103b875a#diff-cc015bc94e16ef0a22a18fb5bec949b1eb4ddca6326461e44289d98ab9aa17e5R509

**Pull request merged**